CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- On Windows, fix applications not exiting gracefully.
```

Comment:
I think it's worth mentioning the use-after-free (and perhaps how window destruction triggers it) here.

Reply:
How about:
"On Windows, fix applications not exiting gracefully due to `thread_event_target_callback` accessing corrupted memory."
or a more details:
"On Windows, fix applications not exiting gracefully due to `WM_DESTROY` events destroying `subclass_input` but `WM_NCDESTOY` is send after `WM_DESTROY` where the former uses `subclass_input."

Reply:
changed it to my first suggestion :)